data/README.md

```diff
@@ -0,0 +1,39 @@
+# Data sets and directories
+Each directory is a separate dataset.
+The directory structure is as follows:
+
+
+## 24hr 4 channel data
+These datasets contain 24 hour timelapse data with 4 channels at 10 minute intervals.
+
+#### Metadata: [metadata_24h.csv](metadata_24h.csv)
+#### Raw data: [20230920ChromaLiveTL_24hr4ch](20230920ChromaLiveTL_24hr4ch)
+#### Maximal intensity projections: [20230920ChromaLiveTL_24hr4ch_MaxIP](20230920ChromaLiveTL_24hr4ch_MaxIP)
```

Comment:
Consider updating for formatting for these (and others like it in this file). Generally, these looked like bolded list items instead of section headers (just one interpretation, so I could be wrong here). While the display of the content looks legible, it could be important to consider web accessibility (and related publication possibilities) as part of this.